proto/cosmos/accounts/testing/rotation/v1/partial.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
```

Comment:
this is the proto file for a testing x/account meant to test account abstraction.

added pub key rotation to make things more spicy